cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/transformers/CgmesRatioTapChangerBuilder.java

```diff
@@ -39,14 +40,27 @@ protected void addRegulationData() {
     protected void addSteps() {
         String tableId = p.getId(CgmesNames.RATIO_TAP_CHANGER_TABLE);
         if (tableId != null) {
-            addStepsFromTable(tableId);
+            PropertyBags table = context.ratioTapChangerTable(tableId);
+            if (table == null) {
+                addStepsFromLowHighIncrement();
+                return;
+            }
+            int min = table.stream().map(p -> p.asInt(CgmesNames.STEP)).min(Integer::compareTo).orElseThrow(() -> new PowsyblException("Should at least contain one step"));
```

Comment:
Just to be sure to understand well: if we have a table defined by `highStep`, `lowStep` and `voltageStepIncrement`, and if we also have a table, we check that the table is coherent with the increment definition before importing the data from the table. What is the criteria to define that coherence? The index? It should not be the size of the table?

Reply:
We check that there is no "hole" in the table i.e. that the number of steps is continuous to chose if we keep or not the table. For example, if we have a table with a step 1, a step 3 but no step 2, we do not keep it. We keep however the table even if it has less steps than what the parameter `highStep` suggests.

Reply:
Maybe, we can preserve the values of the table and only complete linearly the holes on the table. Try to use the table as much as possible.  If you want we can do it.

Reply:
I don't think we need to go to that extent.